std_msgs/Header header
float32 amp_hours
float32 amp_hours_charged